docs/README.md

```diff
@@ -25,3 +25,11 @@ from the `docs/` folder, that'll be easier.
 ```bash
 alias run_docs='make clean; make html; firefox build/html/index.html'
 ```
+
+
+### Writing good docstrings
+
+- Start with [RST and Sphinx CheatSheet](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html)
+- Linking to method in a class `:func:~mymodule.myclass.myfunc`.
```

Comment:
That would be `:meth` actually, see here https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects (you can omit the `:py:`)

Reply:
But I think you can use any tag (role) and it works anyways